pkg/config/analysis/analyzers/envoyfilter/envoyfilter.go

```diff
@@ -71,7 +139,30 @@ func (*EnvoyPatchAnalyzer) analyzeEnvoyFilterPatch(r *resource.Instance, c analy
 				}
 
 				c.Report(collections.IstioNetworkingV1Alpha3Envoyfilters.Name(), message)
+			} else if patch.Patch.Operation == network.EnvoyFilter_Patch_MERGE {
+				count := 0
+				for _, name := range patchFilterNames {
+					if instanceName.String() == name {
+						count++
+						break
+					}
+				}
+				if count > 0 && ef.Priority == 0 {
+					// there is more than one envoy filter using the same name with the REPLACE operation
+					// provide a warning indicating that no priority was set and a relative operation was used
+
+					message := msg.NewEnvoyFilterUsesRelativeOperation(r)
+
+					if line, ok := util.ErrorLine(r, fmt.Sprintf(util.EnvoyFilterConfigPath, index)); ok {
+						message.Line = line
+					}
+
+					c.Report(collections.IstioNetworkingV1Alpha3Envoyfilters.Name(), message)
+				}
```

Comment:
These same lines are repeated in 2 places, so also would be good to move to a function?